src/Symfony/Bridge/Propel1/phpunit.xml.dist

```diff
@@ -7,9 +7,7 @@
          bootstrap="vendor/autoload.php"
 >
     <php>
-        <!-- Disable E_USER_DEPRECATED until 3.0 -->
-        <!-- php -r 'echo -1 & ~E_USER_DEPRECATED;' -->
-        <ini name="error_reporting" value="-16385"/>
+        <ini name="error_reporting" value="-1" />
```

Comment:
I suggest removing this entirely from the phpunit config (as done before the change avoiding deprecation failures). What do you think ?